tests/test_number.py

```diff
@@ -170,5 +175,5 @@ def test_scientific(test_args, expected):
         ([1, humanize.intword, 1e6, None, "under "], "under 1.0 million"),
     ],
 )
-def test_clamp(test_args, expected):
+def test_clamp(test_args: list[typing.Any], expected: str) -> None:
```

Comment:
For nicer type hints, you can put

```
if typing.TYPE_CHECKING:
    from typing import Any
```
at the top of the file, and just go `list[Any]` here.

But that's a stylistic choice, nothing wrong with this either.